src/test/scala/com/microsoft/ml/spark/lightgbm/split1/VerifyLightGBMClassifier.scala

```diff
@@ -133,7 +133,14 @@ class VerifyLightGBMClassifier extends Benchmarks with EstimatorFuzzing[LightGBM
       .fit(df)
       .transform(df).drop(categoricalColumns: _*)
       .withColumnRenamed("c_y", labelCol)
-    LightGBMUtils.getFeaturizer(df2, labelCol, featuresCol).transform(df2)
+    df2
+  }.cache()
+  lazy val indexedBankTrainDF: DataFrame = {
+    LightGBMUtils.getFeaturizer(unfeaturizedBankTrainDF, labelCol, featuresCol,
```

Comment:
any reason you created a static method (LightGBMUtils.getFeaturizer) instead of exposing a class? Is this still composable as pipeline?

Reply:
that is just a helper method that sets up the parameters and then calls the Featurize estimator:
https://github.com/Azure/mmlspark/blob/05a2bef54fa88a2293020215cf4cae34f2d212c5/src/main/scala/com/microsoft/ml/spark/lightgbm/LightGBMUtils.scala#L51
so I could just use the class but this seems like better code re-use